demo-app/src/app/validation/validation-playground/markdown/component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, OnInit, Renderer } from '@angular/core';
 import { AbstractControl, FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
 
-import { ActivitySector, Province, User } from '../models';
+import { ActivitySector, Province, User } from '../../models';
```

Comment:
this is a copy-paste from https://github.com/ofrogon/ng2-materialize/blob/86570d326253ec76a2ffae20fc36bd185f9f51ca/demo-app/src/app/validation/validation-playground/validation-playground.component.ts .. it should stay `'../models';`.